R/tokenizers.R

```diff
@@ -228,3 +229,71 @@ normalize_characters <- function(x) {
 
     return(x)
 }
+
+#' @rdname tokenize_internal
+#' @export
+tokenize_word4 <- function(x, split_hyphens = FALSE, split_tags = FALSE, split_elisions = FALSE,
+                           verbose = quanteda_options("verbose"), ...) {
+    
+    rules <- quanteda::data_breakrules_word
```

Comment:
I think there should still be options to enable/disable rules when calling the function, without modying a global object. For example, when analyzing french and english texts in a same R session, I'd like to enable the elision split only for french.

Reply:
I created the data object only to avoid reading custom.yml from the disk every time we run the tokenizer. It should be more easily customizable by the users admittedly.

Please note that the role of the custom RBBI shrunk in my latest commits (URLs and emails are also restored using `tokens_restore()` to leave @ and # in them intact). RBBI is used mainly for hyphens and elisions now.